### ISSUE TITLE
S3DIS dataset Instance label mismatch

### ISSUE THREAD

**Issue by perrydoremi** [oneformer3d/oneformer3d#28]
Thanks for this amazing work and contribution to the 3D unified segmentation.
I noticed there might be a instance label mismatch for S3DIS dataset implementation.

I prepare the data following the original [instruction](https://github.com/open-mmlab/mmdetection3d/tree/22aaa47fdb53ce1870ff92cb7e3f96ae38d17f61/data/s3dis) in mmdetection3D.

Then I use `python tools/train.py configs/oneformer3d_1xb2_s3dis-area-5.py` to train S3DIS, and everything went smoothly, however, I noticed that the instance counting msg for the dataset seems wrong (I copy paste one clip from the terminal here):
> 03/03 21:00:27 - mmengine - INFO - ------------------------------ 
> 03/03 21:00:27 - mmengine - INFO - The length of training dataset: 44
> 03/03 21:00:27 - mmengine - INFO - The number of instances per category in the dataset:
> +----------+--------+
> | category | number | 
> +----------+--------+
> | ceiling  | 69     |
> | floor    | 155    |
> | wall     | 6      |
>| beam     | 90     |
>| column   | 27     |
>| window   | 0      |
>| door     | 0      |
>| table    | 0      |
>| chair    | 0      |
>| sofa     | 0      |
>| bookcase | 0      |
>| board    | 0      |
>| clutter  | 0      |
>+----------+--------+

For the above counting, the "thing" instances (semantic class [7, 8, 9, 10, 11]) seem to be miscounted to "stuff" (semantic class [0, 1, 2, 3, 4, 5, 6, 12])

And I sourced a little bit, It turned out that when you preparing S3DIS data using 
`python tools/create_data.py s3dis --root-path ./data/s3dis --out-dir ./data/s3dis --extra-tag s3dis`,
In [s3dis_data_util.py](https://github.com/open-mmlab/mmdetection3d/blob/22aaa47fdb53ce1870ff92cb7e3f96ae38d17f61/tools/dataset_converters/s3dis_data_utils.py) line 28-32. "Thing" semantic labels [7, 8, 9, 10, 11] are mapped to [0, 1, 2, 3, 4], which are then assigned to that bounding box as the ground truth instance label. Therefore, the msg I show above for S3DIS may counted the instances of 'table', 'chair', 'sofa', 'bookcase', 'board' [7, 8, 9, 10, 11] as 'ceiling', 'floor', 'wall', 'beam', 'column' [0, 1, 2, 3, 4]

If I understood this correctly or I missed something?

**Comment by filaPro** [comment]
Hi @perrydoremi ,
I think it's just a bug with this log message. And actually these classes are presented in annotations. I've checked the log file that we are providing in the readme, and it also have these zeros for last 8 classes. So if you do not have other errors, you can ignore this message.
